utils/oscap-xccdf.c

```diff
@@ -1041,6 +1046,9 @@ bool getopt_xccdf(int argc, char **argv, struct oscap_action *action)
                 } else {
                     action->f_ovals = NULL;
                 }
+
+		if (action->rule != NULL && action->profile == NULL)
+			return oscap_module_usage(action->module, stderr, "Profile needs to be specified when evaluating a single rule!");
```

Comment:
It doesn't evaluate anything because `default` profile has no rules selected.
I guess the question is whether the one rule cannot be evaluated as if part of the `default` profile, when no profile is provided.